### PR TITLE
Split example workflows into tutorial and examples section

### PR DESCRIPTION
**Pull request by kenxu95**
Task: https://linear.app/aqueducthq/issue/ENG-1546/split-notebooks-into-beginneradvanced-concepts

The notebook files changes were made from https://github.com/aqueducthq/aqueduct/pull/575 in the Aqueduct repo.

There's a bunch of stuff I noticed that could be improved, but was out of scope of this task. Noted those here: https://linear.app/aqueducthq/issue/ENG-1820/make-improvements-to-our-internal-example-notebooks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,13 +67,18 @@ For more on this pipeline, check our [Quickstart Guide](quickstart-guide.md).
 * [Artifacts](artifacts.md)
 * [Metrics & Checks](metrics-and-checks.md)
 
+### Tutorial Workflows
```
Comment (vsreekanti) [inline README.md:70]: Tiny nit: Let's just say "Tutorials" and "Examples" as the headings.

File: example-workflows/README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
 # Example Workflows
 
+Here are some tutorial notebooks showcasing the various features of Aqueduct:
+
+* [Quickstart](example-workflows/quickstart-tutorial.md)
```
Comment (vsreekanti) [inline example-workflows/README.md:5]: As I mentioned on the other PR, I'd prefer to just keep one quickstart rather than have two.

**Review by vsreekanti — APPROVED** [review]
LGTM, @kenxu95! Couple comments inline. Should be good to go once we resolve the quickstart duplication.

File: example-workflows/README.md (hunk 2 above)
Comment (kenxu95) [reply example-workflows/README.md:5]: Updated the quickstart guide and translated it into a notebook format here.

**Comment by kenxu95** [comment]
oh oops, I accidentally request review again. Can ignore.

**Pull request merged**